.golangci.yml

```diff
@@ -66,5 +67,10 @@ linters-settings:
     checks:
       - "all"
       - "-SA1019"   # TODO(fix) Using a deprecated function, variable, constant or field
-
-
+  depguard:
+    rules:
+      main:
+        list-mode: strict
+        deny:
+          - pkg: "k8s.io/klog"
```

Comment:
looks like this is prefix match, is there a option

Reply:
Allow is a prefix of packages to allow. A dollar sign $ can be used at the end of a package to specify it must be exact match only.

Deny is a map where the key is a prefix of the package to deny, and the value is a suggestion on what to use instead. A dollar sign $ can be used at the end of a package to specify it must be exact match only.

A Prefix List just means that a package will match a value, if the value is a prefix of the package. Example github.com/OpenPeeDeeP/depguard package will match a value of github.com/OpenPeeDeeP but won't match github.com/OpenPeeDeeP/depguard/v2.

https://github.com/OpenPeeDeeP/depguard

Reply:
Thanks for your guidance. It works now.